### ISSUE TITLE
Add table of sections to the readme file

### ISSUE THREAD

**Issue by T-Fathima** [BeehiveCommunity/Frontend_Mentor#470]
### Description

The current README file lacks a Table of Contents, which makes it difficult for users to navigate through the document. Adding a Table of Contents will help improve accessibility and readability, especially for larger README files with multiple sections.

### Checklist

- [X] I have checked the existing issues
- [X] I have read the [Contributing Guidelines](https://github.com/beRajeevKumar/Frontend_Mentor/blob/main/Code_Of_Conduct.md)
- [X] I'm a GSSoC'24-Extd contributor
- [X] I'm a Hacktoberfest'24 contributor
- [X] I am willing to work on this issue (optional)
